examples/currency-transfer.yaml

```diff
@@ -0,0 +1,71 @@
+
+'@context':
+  - https://git.io/vf-examples-jsonld-context
+  - bob: https://bob.example/
+    alice: https://alice.example/
+    lets: https://lets.example/
+
+'@id': rgh:valueflows/valueflows/master/examples/currency-transfer.yaml
+'@graph':
+
+  # accounts before
+
+  - '@id': lets:b75d8f6a-e2df-4e52-b36d-1a22a66f4ead
+    '@type': EconomicResource
+    skos:note: Alice's account 
+    classifiedAs: wd:Q179941
+    currentQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: -200
+
+  - '@id': lets:3be5259d-10f0-431c-9fec-9c0c15a461d3
+    '@type': EconomicResource
+    skos:note: Bob's account 
+    classifiedAs: wd:Q179941
+    currentQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 500
+
+  - '@id': lets:e0caf1e3-4a9b-4927-a19c-3f681d234529
+    '@type': Process
+    skos:note: moving credits between accounts
+
+  - '@id': lets:0468e9b9-4d54-4eee-9d3f-bc76db11710a
+    '@type': EconomicEvent
+    action: decrement
+    inputOf: lets:e0caf1e3-4a9b-4927-a19c-3f681d234529
+    involves: lets:b75d8f6a-e2df-4e52-b36d-1a22a66f4ead
+    involvesQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 100
+
+  - '@id': lets:33f11508-92b3-4095-9e03-6678cddf7adf
+    '@type': EconomicEvent
+    action: increment
+    outputOf: lets:e0caf1e3-4a9b-4927-a19c-3f681d234529
+    involves: lets:3be5259d-10f0-431c-9fec-9c0c15a461d3
+    involvesQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 100
+    transfer:
+      - rights: control
+        provider: https://alice.example/
+        recipient: https://bob.example/
```

Comment:
This is an example place where it is not clear what Alice is giving up some rights to.  SInce this is simple, you could figure it out, and a machine could figure it out, although it would be a pain.  But if there were a service created, and a transfer fee, and whatever, it would start to become even more of a pain.

Reply:
I think we only transfer R&R to products not services (including usage and work). In my examples transfer always happens over `involvesQuantity` of a resource specified either by `involves` or classification or specification.

Reply:
It doesn't matter, just trying to say it is not straight-forward to find the other resource in all cases.  Maybe the wrong place to make the comment since this is probably the simplest example.